_install-and-configure/upgrade-opensearch/index.md

```diff
@@ -137,6 +137,31 @@ td {
         <th>OpenSearch Version</th>
         <th>Elasticsearch Version</th>
     </tr>
+    <tr>
+        <td>9.10.0</td>
+        <td>2.13.0</td>
+        <td>8.13</td>
+    </tr>
+    <tr>
+        <td>9.8.0</td>
+        <td>2.12.0</td>
+        <td>8.11.0</td>
+    </tr>
+    <tr>
+        <td>9.7.0</td>
+        <td>2.11.1<br>2.9.0</td>
```

Comment:
Do we want to include 2.10, too?

I guess if 2.9 and 2.11 use the same version of Lucene, it would have been pretty weird for 2.10 to go off and use something else.

Reply:
Maybe we update these (and older ones) to use a hyphen so it's more clear it's a range.